irc/server.go

```diff
@@ -1061,13 +1061,17 @@ func (server *Server) DeleteMessage(target, msgid, accountName string) (err erro
 				if status, _, _ := channel.historyStatus(config); status == HistoryEphemeral {
 					hist = &channel.history
 				}
+			} else {
+				return errNoSuchChannel
```

Comment:
I'm interested in your feedback on what errors should be thrown here. and below.